vendor-bundles.webpack.config.js

```diff
@@ -37,6 +40,10 @@ let config = {
     devtool : 'source-map'
 };
 
+config.resolve = {
+    modules: [path.join(__dirname, "src", "common"), "node_modules"]
```

Comment:
i doubt you need __dirname here.   if we needed it, wouldn't it be necessary for node_modules as well?

Reply:
Ok, removed